captum/optim/_utils/circuits.py

```diff
@@ -0,0 +1,63 @@
+from typing import Optional, Tuple, Union
+
+import torch
+import torch.nn as nn
+
+from captum.optim._param.image.transform import center_crop_shape
+from captum.optim._utils.models import collect_activations
+from captum.optim._utils.typing import ModelInputType, TransformSize
+
+
+def get_expanded_weights(
+    model,
+    target1: nn.Module,
+    target2: nn.Module,
+    crop_shape: Optional[Union[Tuple[int, int], TransformSize]] = None,
+    model_input: ModelInputType = torch.zeros(1, 3, 224, 224),
+) -> torch.Tensor:
+    """
+    Extract meaningful weight interactions from between neurons which aren’t
+    literally adjacent in a neural network, or where the weights aren’t directly
+    represented in a single weight tensor.
+    Schubert, et al., "Visualizing Weights", Distill, 2020.
+    See: https://distill.pub/2020/circuits/visualizing-weights/
+
+    Args:
+        model:  PyTorch model instance.

```

Comment:
nit: type hint: (nn.Module)
https://github.com/pytorch/captum/blob/master/captum/attr/_core/deep_lift.py#L112

Reply:
I've added the type hint!